GCO-2381 Search input for TupleList (optional) (#51)
dotwebstack/dotwebstack-webcomponents: docs/api.md

The column on which the list will be sorted initially. The value is an array with exactly 2 elements (tuple).
First element is the column name, second element is a boolean whether the column is sorted in ascending order or not.

### `suggest`: SuggestProps (optional)

> This property is used in combination with the search property. When the search property is provided a user get's a search

Reviewer: ```suggestion
This property is used in combination with the search property. When the search property is provided a user gets a search
```

input in order to filter the TupleList result. Use the *suggestions* [TupleResult, string] property of the SuggestProps to provide suggestions
based on the result and the provided column name (should be a field which is also used in search to provide consistency)
When using suggestions it is best practice to use the same field (and only that field) in the search prop.

### `search`: SearchListProps
The list will be sorted while typing when instant search is true (use when no suggestions are enabled, 
set to false when suggestions are provided)

> By default search will search on all names in the TupleResult, you can provide a list to on which fields the search is provided

Reviewer: Deze zin is onduidelijk, zoiets?
```suggestion
Search will use all names in the TupleResult by default, or a custom list if provided.
```
